src/components/Filter/Filter.js

```diff
@@ -16,12 +16,14 @@ export default class Filter extends Component {
 
   render() {
     return (
-     <select name='selection' onChange={this.handleChange}>
-      <option value=''>Filter movies by...</option>
-      <option value='alphabetically'>Alphabetically</option>
-      <option value='rating'>Rating</option>
-      <option value='release'>Release</option>
-     </select>
+    <span className="filter">
+      <select  name='selection' onChange={this.handleChange}>
+        <option value=''>Filter movies by...</option>
+        <option value='alphabetically'>Alphabetically</option>
+        <option value='rating'>Rating</option>
+        <option value='release'>Release</option>
+      </select>
+    </span>
```

Comment:
I'm curious to know why this was wrapped in a `<span>` rather than changing the .CSS to `display: inline` or some other solutions, either way works but it could make the HTML a bit less readable